examples/taskworkflow.md

```diff
@@ -0,0 +1,87 @@
+### About
+The `taskworkfow.go` file contains code to pull and unpack a container image
+using the `naive` snaphotter, create a container and execute a task using the
+`firecracker-containerd` runtime.
+
+### Building
+After checking out the `firecracker-containerd` repo, you can build this
+example using the `go build` command. Example:
+```bash
+$ go build -o /tmp/taskworkflow examples/taskworkflow.go
+```
+
+### Running
+For a basic workflow, without any networking setup, just run the executable
+built with the command in the previous section as super user:
+```bash
+$ sudo /tmp/taskworkflow
+```
+
+For a workflow with networking setup for the container, create a tap device
+for the VM by following [these instructions](https://github.com/firecracker-microvm/firecracker/blob/master/docs/network-setup.md).
+
+This will create a tap device named `tap0`. Now, edit the kernel args in the
```

Comment:
Wow we really need to fix this problem.  https://github.com/firecracker-microvm/firecracker-containerd/issues/76 is the tracking issue.

Reply:
It's a bit circular. If y'all are fine with this PR, it's going to turn into a very simple matter of adding more fields in the protobuf and processing unmarshalled fields :)

Reply:
Yep!